src/core/algorithms/gfd/gfd.h

```diff
@@ -43,5 +51,9 @@ class Gfd {
         conclusion_ = conclusion;
     }
 
+    bool operator==(Gfd const& gfd) const;
+
+    bool operator!=(Gfd const& gfd) const;
+
     std::string ToString();
-};
+};
```

Comment:
Add blank line at the end. Also applies to other files

Reply:
I think the CI check complains when I do this